### ISSUE TITLE
VS Code crashes when certain text is pasted in

### ISSUE THREAD

**Issue by nexxai** [microsoft/vscode#111479]
- VSCode Version: October 2020 (1.51)
- OS Version: macOS Big sur 11.0.1

Steps to Reproduce:

1. Paste the following text into an editor window:

𝙣𝙖𝙥-𝙝𝙤𝙝𝙤𝙝𝙤-𝙡𝙚𝙤𝙣 𝙫𝙚𝙜𝙚𝙢𝙞𝙩𝙚

2. Get ready to be extremely fast when you open up VSCode again because it will keep crashing until you manage to close the open file with that text in it (my suggestion on macOS at least: use CMD-[SPACE] to bring up the launcher, open VS Code, and have your mouse pointer already in position over the 'x' of the tab that has the text in it.

Does this issue occur when all extensions are disabled?: Yes

**Comment by egamma** [comment]
This happens on Big Sur assigning to @isidorn since I think you are on Big Sur.
I cannot reproduce on Catalina.

**Comment by isidorn** [comment]
This works just fine for me.
@nexxai can you reproduce with vscode insiders https://code.visualstudio.com/insiders/

**Comment by nexxai** [comment]
Can confirm that Insiders does not crash for me.  It's strange because it is even happening (in non-Insiders) when I run it with --disable-extensions.  Does the app write any logs anywhere that I can look into for further details?

**Comment by isidorn** [comment]
Might be due to electron, but if you can not reproduce with Insiders I would believe this is fixed with latest and I would just close this. However first forwarding to @deepak1556 so he can decide

**Comment by deepak1556** [comment]
Can you start with `code --crash-reporter-directory <absolute-path>` and once crash happens, attach the generated crash dump here. Thanks!

**Comment by nexxai** [comment]
Attached: [7d83fbe1-13b9-4407-b80f-ec0c48577836.dmp.zip](https://github.com/microsoft/vscode/files/5633397/7d83fbe1-13b9-4407-b80f-ec0c48577836.dmp.zip)

**Comment by deepak1556** [comment]
Symbolicated trace

```
Symbolicating...
Operating system: Mac OS X
                  10.16.0 20B29
CPU: amd64
     family 6 model 126 stepping 5
     8 CPUs

GPU: UNKNOWN

Crash reason:  EXC_BAD_INSTRUCTION / EXC_I386_INVOP
Crash address: 0x7fff22ff5adc
Process uptime: 34 seconds

Thread 0 (crashed)
 0  AppKit!-[NSApplication _crashOnException:] + 0x15b
    rax = 0x00007fe8b5066e00   rdx = 0x0000000000000001
    rcx = 0xfa01605160a200f3   rbx = 0x00007fff201d6d00
    rsi = 0x0000000000000631   rdi = 0x00007ffee8f61c10
    rbp = 0x00007ffee8f620a0   rsp = 0x00007ffee8f62070
     r8 = 0x0000000000000600    r9 = 0x0000000000000000
    r10 = 0x0000000000000031   r11 = 0x00007ffee8f61772
    r12 = 0x00007fff802d2b50   r13 = 0x00007fe8b478d5f0
    r14 = 0x00007fe8bae30e30   r15 = 0x00007fff7c551aef
    rip = 0x00007fff22ff5adc
    Found by: given as instruction pointer in context
 1  AppKit!-[NSApplication reportException:] + 0x2a5
    rbp = 0x00007ffee8f62200   rsp = 0x00007ffee8f620b0
    rip = 0x00007fff22ff57ef
    Found by: previous frame's frame pointer
 2  AppKit!uncaughtErrorProc + 0x91
    rbp = 0x00007ffee8f62230   rsp = 0x00007ffee8f62210
    rip = 0x00007fff230a833e
    Found by: previous frame's frame pointer
 3  CoreFoundation!__handleUncaughtException + 0x2cc
    rbp = 0x00007ffee8f622e0   rsp = 0x00007ffee8f62240
    rip = 0x00007fff20528826
    Found by: previous frame's frame pointer
 4  libobjc.A.dylib!_objc_terminate() + 0x5a
    rbp = 0x00007ffee8f62300   rsp = 0x00007ffee8f622f0
    rip = 0x00007fff201e69ab
    Found by: previous frame's frame pointer
 5  libc++abi.dylib!std::__terminate(void (*)()) + 0x8
    rbp = 0x00007ffee8f62320   rsp = 0x00007ffee8f62310
    rip = 0x00007fff202fe847
    Found by: previous frame's frame pointer
 6  libc++abi.dylib!__cxxabiv1::failed_throw(__cxxabiv1::__cxa_exception*) + 0x1b
    rbp = 0x00007ffee8f62340   rsp = 0x00007ffee8f62330
    rip = 0x00007fff20301167
    Found by: previous frame's frame pointer
 7  libc++abi.dylib!__cxa_throw + 0x71
    rbp = 0x00007ffee8f62370   rsp = 0x00007ffee8f62350
    rip = 0x00007fff2030112e
    Found by: previous frame's frame pointer
 8  libobjc.A.dylib!objc_exception_throw + 0x15e
    rbp = 0x00007ffee8f63350   rsp = 0x00007ffee8f62380
    rip = 0x00007fff201e44f7
    Found by: previous frame's frame pointer
 9  CoreFoundation!+[NSException raise:format:arguments:] + 0x58
    rbp = 0x00007ffee8f63370   rsp = 0x00007ffee8f63360
    rip = 0x00007fff204d5512
    Found by: previous frame's frame pointer
10  Foundation!-[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 0xbf
    rbp = 0x00007ffee8f63470   rsp = 0x00007ffee8f63380
    rip = 0x00007fff2128e6c9
    Found by: previous frame's frame pointer
11  AppKit!-[NSWindow setTitle:] + 0x8e
    rbp = 0x00007ffee8f634b0   rsp = 0x00007ffee8f63480
    rip = 0x00007fff22c94a75
    Found by: previous frame's frame pointer
12  Electron Framework!electron::NativeWindowMac::SetTitle(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) [native_window_mac.mm : 1023 + 0x13]
    rbp = 0x00007ffee8f634d0   rsp = 0x00007ffee8f634c0
    rip = 0x000000010d2c201f
    Found by: previous frame's frame pointer
13  Electron Framework!gin_helper::Dispatcher<void (electron::api::TopLevelWindow*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)>::DispatchToCallback(v8::FunctionCallbackInfo<v8::Value> const&) [callback.h : 132 + 0x3]
    rbp = 0x00007ffee8f63560   rsp = 0x00007ffee8f634e0
    rip = 0x000000010d252e23
    Found by: previous frame's frame pointer
14  Electron Framework!v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [api-arguments-inl.h : 158 + 0x6]
    rbp = 0x00007ffee8f63610   rsp = 0x00007ffee8f63570
    rip = 0x000000010e7ca79f
    Found by: previous frame's frame pointer
15  Electron Framework!v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [builtins-api.cc : 111 + 0xb]
    rbp = 0x00007ffee8f636c0   rsp = 0x00007ffee8f63620
    rip = 0x000000010e7c9cc1
    Found by: previous frame's frame pointer
16  Electron Framework!v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [builtins-api.cc : 141 + 0x21]
    rbp = 0x00007ffee8f63750   rsp = 0x00007ffee8f636d0
    rip = 0x000000010e7c9352
    Found by: previous frame's frame pointer
17  Electron Framework!Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit + 0x38
    rbp = 0x00007ffee8f63778   rsp = 0x00007ffee8f63760
    rip = 0x000000010f1fb7d8
    Found by: previous frame's frame pointer
18  Electron Framework!Builtins_InterpreterEntryTrampoline + 0xd5
    rbp = 0x00007ffee8f63828   rsp = 0x00007ffee8f63788
    rip = 0x000000010f18f475
    Found by: previous frame's frame pointer
19  Electron Framework!Builtins_ArgumentsAdaptorTrampoline + 0xbf
    rbp = 0x00007ffee8f63870   rsp = 0x00007ffee8f63838
    rip = 0x000000010f188ebf
    Found by: previous frame's frame pointer
20  Electron Framework!Builtins_InterpreterEntryTrampoline + 0xd5
    rbp = 0x00007ffee8f63980   rsp = 0x00007ffee8f63880
    rip = 0x000000010f18f475
    Found by: previous frame's frame pointer
21  Electron Framework!Builtins_ArgumentsAdaptorTrampoline + 0xbf
    rbp = 0x00007ffee8f639c0   rsp = 0x00007ffee8f63990
    rip = 0x000000010f188ebf
    Found by: previous frame's frame pointer
22  Electron Framework!Builtins_JSEntryTrampoline + 0x5a
    rbp = 0x00007ffee8f63a08   rsp = 0x00007ffee8f639d0
    rip = 0x000000010f18cfba
    Found by: previous frame's frame pointer
23  Electron Framework!Builtins_JSEntry + 0x78
    rbp = 0x00007ffee8f63a70   rsp = 0x00007ffee8f63a18
    rip = 0x000000010f18cd98
    Found by: previous frame's frame pointer
24  Electron Framework!v8::internal::(anonymous namespace)::Invoke(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) [simulator.h : 142 + 0x8]
    rbp = 0x00007ffee8f63bc0   rsp = 0x00007ffee8f63a80
    rip = 0x000000010e8870fc
    Found by: previous frame's frame pointer
25  Electron Framework!v8::internal::Execution::Call(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int, v8::internal::Handle<v8::internal::Object>*) [execution.cc : 461 + 0x8]
    rbp = 0x00007ffee8f63c40   rsp = 0x00007ffee8f63bd0
    rip = 0x000000010e886546
    Found by: previous frame's frame pointer
26  Electron Framework!v8::Function::Call(v8::Local<v8::Context>, v8::Local<v8::Value>, int, v8::Local<v8::Value>*) [api.cc : 4995 + 0x15]
    rbp = 0x00007ffee8f63d60   rsp = 0x00007ffee8f63c50
    rip = 0x000000010e7781a8
    Found by: previous frame's frame pointer
27  Electron Framework!node::InternalMakeCallback(node::Environment*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*, node::async_context) [callback.cc : 156 + 0xe]
    rbp = 0x00007ffee8f63e20   rsp = 0x00007ffee8f63d70
    rip = 0x0000000112c913dc
    Found by: previous frame's frame pointer
28  Electron Framework!node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*, node::async_context) [callback.cc : 222 + 0x1f]
    rbp = 0x00007ffee8f63e80   rsp = 0x00007ffee8f63e30
    rip = 0x0000000112c91641
    Found by: previous frame's frame pointer
29  Electron Framework!gin_helper::internal::CallMethodWithArgs(v8::Isolate*, v8::Local<v8::Object>, char const*, std::__1::vector<v8::Local<v8::Value>, std::__1::allocator<v8::Local<v8::Value> > >*) [event_emitter_caller.cc : 23 + 0x14]
    rbp = 0x00007ffee8f63ec0   rsp = 0x00007ffee8f63e90
    rip = 0x000000010d33789f
    Found by: previous frame's frame pointer
30  Electron Framework!v8::Local<v8::Value> gin_helper::EmitEvent<base::BasicStringPiece<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, v8::Local<v8::Object>&, std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&>(v8::Isolate*, v8::Local<v8::Object>, base::BasicStringPiece<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > const&, v8::Local<v8::Object>&, std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&) [event_emitter_caller.h : 50 + 0x17]
    rbp = 0x00007ffee8f63f20   rsp = 0x00007ffee8f63ed0
    rip = 0x000000010d276cb8
    Found by: previous frame's frame pointer
31  Electron Framework!bool gin_helper::EventEmitter<electron::api::WebContents>::EmitWithEvent<std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&>(base::BasicStringPiece<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, v8::Local<v8::Object>, std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&) [event_emitter.h : 114 + 0x11]
    rbp = 0x00007ffee8f63fa0   rsp = 0x00007ffee8f63f30
    rip = 0x000000010d276bbe
    Found by: previous frame's frame pointer
32  Electron Framework!bool gin_helper::EventEmitter<electron::api::WebContents>::Emit<std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&>(base::BasicStringPiece<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> >&, bool&) [event_emitter.h : 79 + 0x17]
    rbp = 0x00007ffee8f64000   rsp = 0x00007ffee8f63fb0
    rip = 0x000000010d2694ec
    Found by: previous frame's frame pointer
33  Electron Framework!electron::api::WebContents::TitleWasSet(content::NavigationEntry*) [electron_api_web_contents.cc : 1436 + 0xd]
    rbp = 0x00007ffee8f64120   rsp = 0x00007ffee8f64010
    rip = 0x000000010d2693ac
    Found by: previous frame's frame pointer
34  Electron Framework!content::WebContentsImpl::UpdateTitleForEntry(content::NavigationEntry*, std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> > const&) [web_contents_impl.cc : 5301 + 0xc]
    rbp = 0x00007ffee8f64190   rsp = 0x00007ffee8f64130
    rip = 0x000000010e451115
    Found by: previous frame's frame pointer
35  Electron Framework!content::RenderFrameHostImpl::UpdateTitle(base::Optional<std::__1::basic_string<unsigned short, base::string16_internals::string16_char_traits, std::__1::allocator<unsigned short> > > const&, base::i18n::TextDirection) [render_frame_host_impl.cc : 3589 + 0x13]
    rbp = 0x00007ffee8f641f0   rsp = 0x00007ffee8f641a0
    rip = 0x000000010e131040
    Found by: previous frame's frame pointer
36  Electron Framework!blink::mojom::LocalFrameHostStubDispatch::Accept(blink::mojom::LocalFrameHost*, mojo::Message*) [frame.mojom.cc : 4174 + 0x9]
    rbp = 0x00007ffee8f64410   rsp = 0x00007ffee8f64200
    rip = 0x000000010dd3b796
    Found by: previous frame's frame pointer
37  Electron Framework!mojo::InterfaceEndpointClient::HandleValidatedMessage(mojo::Message*) [interface_endpoint_client.cc : 554 + 0x9]
    rbp = 0x00007ffee8f644a0   rsp = 0x00007ffee8f64420
    rip = 0x000000010fa269e6
    Found by: previous frame's frame pointer
38  Electron Framework!mojo::MessageDispatcher::Accept(mojo::Message*) [message_dispatcher.cc : 46 + 0x9]
    rbp = 0x00007ffee8f644e0   rsp = 0x00007ffee8f644b0
    rip = 0x000000010fa28c76
    Found by: previous frame's frame pointer
39  Electron Framework!IPC::(anonymous namespace)::ChannelAssociatedGroupController::AcceptOnProxyThread(mojo::Message) [ipc_mojo_bootstrap.cc : 933 + 0xb]
    rbp = 0x00007ffee8f64520   rsp = 0x00007ffee8f644f0
    rip = 0x000000010fcb7b05
    Found by: previous frame's frame pointer
40  Electron Framework!base::internal::Invoker<base::internal::BindState<void (IPC::(anonymous namespace)::ChannelAssociatedGroupController::*)(mojo::Message), scoped_refptr<IPC::(anonymous namespace)::ChannelAssociatedGroupController>, mojo::Message>, void ()>::RunOnce(base::internal::BindStateBase*) [bind_internal.h : 489 + 0x9]
    rbp = 0x00007ffee8f645c0   rsp = 0x00007ffee8f64530
    rip = 0x000000010fcb5b23
    Found by: previous frame's frame pointer
41  Electron Framework!base::TaskAnnotator::RunTask(char const*, base::PendingTask*) [callback.h : 98 + 0x3]
    rbp = 0x00007ffee8f64670   rsp = 0x00007ffee8f645d0
    rip = 0x000000010f70d52f
    Found by: previous frame's frame pointer
42  Electron Framework!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl(base::sequence_manager::LazyNow*, bool*) [thread_controller_with_message_pump_impl.cc : 324 + 0xf]
    rbp = 0x00007ffee8f64740   rsp = 0x00007ffee8f64680
    rip = 0x000000010f71d70a
    Found by: previous frame's frame pointer
43  Electron Framework!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() [thread_controller_with_message_pump_impl.cc : 248 + 0xb]
    rbp = 0x00007ffee8f64790   rsp = 0x00007ffee8f64750
    rip = 0x000000010f71d499
    Found by: previous frame's frame pointer
44  Electron Framework!invocation function for block in base::MessagePumpCFRunLoopBase::RunWorkSource(void*) [message_pump_mac.mm : 499 + 0x6]
    rbp = 0x00007ffee8f647b0   rsp = 0x00007ffee8f647a0
    rip = 0x000000010f76df21
    Found by: previous frame's frame pointer
45  Electron Framework!base::mac::CallWithEHFrame(void () block_pointer) + 0xa
    rbp = 0x00007ffee8f647c0   rsp = 0x00007ffee8f647c0
    rip = 0x000000010f7696ba
    Found by: previous frame's frame pointer
46  Electron Framework!base::MessagePumpCFRunLoopBase::RunWorkSource(void*) [message_pump_mac.mm : 475 + 0x5]
    rbp = 0x00007ffee8f64800   rsp = 0x00007ffee8f647d0
    rip = 0x000000010f76d7ef
    Found by: previous frame's frame pointer
47  CoreFoundation!__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 0x11
    rbp = 0x00007ffee8f64810   rsp = 0x00007ffee8f64810
    rip = 0x00007fff204329fc
    Found by: previous frame's frame pointer
48  CoreFoundation!__CFRunLoopDoSource0 + 0xb4
    rbp = 0x00007ffee8f64840   rsp = 0x00007ffee8f64820
    rip = 0x00007fff20432964
    Found by: previous frame's frame pointer
49  CoreFoundation!__CFRunLoopDoSources0 + 0xf8
    rbp = 0x00007ffee8f648b0   rsp = 0x00007ffee8f64850
    rip = 0x00007fff204326df
    Found by: previous frame's frame pointer
50  CoreFoundation!__CFRunLoopRun + 0x37a
    rbp = 0x00007ffee8f655f0   rsp = 0x00007ffee8f648c0
    rip = 0x00007fff20431111
    Found by: previous frame's frame pointer
51  CoreFoundation!CFRunLoopRunSpecific + 0x233
    rbp = 0x00007ffee8f65690   rsp = 0x00007ffee8f65600
    rip = 0x00007fff204306be
    Found by: previous frame's frame pointer
52  HIToolbox!RunCurrentEventLoopInMode + 0x124
    rbp = 0x00007ffee8f656e0   rsp = 0x00007ffee8f656a0
    rip = 0x00007fff2869cfd0
    Found by: previous frame's frame pointer
53  HIToolbox!ReceiveNextEventCommon + 0x2c5
    rbp = 0x00007ffee8f65760   rsp = 0x00007ffee8f656f0
    rip = 0x00007fff2869cdcc
    Found by: previous frame's frame pointer
54  HIToolbox!_BlockUntilNextEventMatchingListInModeWithFilter + 0x40
    rbp = 0x00007ffee8f65780   rsp = 0x00007ffee8f65770
    rip = 0x00007fff2869caef
    Found by: previous frame's frame pointer
55  AppKit!_DPSNextEvent + 0x373
    rbp = 0x00007ffee8f65b70   rsp = 0x00007ffee8f65790
    rip = 0x00007fff22c49f85
    Found by: previous frame's frame pointer
56  AppKit!-[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 0x556
    rbp = 0x00007ffee8f65de0   rsp = 0x00007ffee8f65b80
    rip = 0x00007fff22c4874b
    Found by: previous frame's frame pointer
57  AppKit!-[NSApplication run] + 0x24a
    rbp = 0x00007ffee8f65e70   rsp = 0x00007ffee8f65df0
    rip = 0x00007fff22c3abda
    Found by: previous frame's frame pointer
58  Electron Framework!base::MessagePumpNSApplication::DoRun(base::MessagePump::Delegate*) [message_pump_mac.mm : 835 + 0xd]
    rbp = 0x00007ffee8f65ed0   rsp = 0x00007ffee8f65e80
    rip = 0x000000010f76e73c
    Found by: previous frame's frame pointer
59  Electron Framework!base::MessagePumpCFRunLoopBase::Run(base::MessagePump::Delegate*) [message_pump_mac.mm : 190 + 0xc]
    rbp = 0x00007ffee8f65f10   rsp = 0x00007ffee8f65ee0
    rip = 0x000000010f76d1a2
    Found by: previous frame's frame pointer
60  Electron Framework!base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run(bool, base::TimeDelta) [thread_controller_with_message_pump_impl.cc : 429 + 0x6]
    rbp = 0x00007ffee8f65f40   rsp = 0x00007ffee8f65f20
    rip = 0x000000010f71dc93
    Found by: previous frame's frame pointer
61  Electron Framework!base::RunLoop::Run() [run_loop.cc : 124 + 0x13]
    rbp = 0x00007ffee8f65fd0   rsp = 0x00007ffee8f65f50
    rip = 0x000000010f6f5a73
    Found by: previous frame's frame pointer
62  Electron Framework!content::BrowserMainLoop::RunMainMessageLoopParts() [browser_main_loop.cc : 1496 + 0x8]
    rbp = 0x00007ffee8f66040   rsp = 0x00007ffee8f65fe0
    rip = 0x000000010dfb5ee3
    Found by: previous frame's frame pointer
63  Electron Framework!content::BrowserMainRunnerImpl::Run() [browser_main_runner_impl.cc : 150 + 0x5]
    rbp = 0x00007ffee8f66060   rsp = 0x00007ffee8f66050
    rip = 0x000000010dfb78f2
    Found by: previous frame's frame pointer
64  Electron Framework!content::BrowserMain(content::MainFunctionParams const&) [browser_main.cc : 47 + 0x6]
    rbp = 0x00007ffee8f660b0   rsp = 0x00007ffee8f66070
    rip = 0x000000010dfb3798
    Found by: previous frame's frame pointer
65  Electron Framework!content::ContentMainRunnerImpl::RunServiceManager(content::MainFunctionParams&, bool) [content_main_runner_impl.cc : 530 + 0x5]
    rbp = 0x00007ffee8f66130   rsp = 0x00007ffee8f660c0
    rip = 0x000000010de5ba66
    Found by: previous frame's frame pointer
66  Electron Framework!content::ContentMainRunnerImpl::Run(bool) [content_main_runner_impl.cc : 879 + 0x10]
    rbp = 0x00007ffee8f661a0   rsp = 0x00007ffee8f66140
    rip = 0x000000010de5b6bb
    Found by: previous frame's frame pointer
67  Electron Framework!service_manager::Main(service_manager::MainParams const&) [main.cc : 454 + 0x9]
    rbp = 0x00007ffee8f664f0   rsp = 0x00007ffee8f661b0
    rip = 0x0000000110eacc18
    Found by: previous frame's frame pointer
68  Electron Framework!content::ContentMain(content::ContentMainParams const&) [content_main.cc : 19 + 0x8]
    rbp = 0x00007ffee8f66580   rsp = 0x00007ffee8f66500
    rip = 0x000000010de5ad84
    Found by: previous frame's frame pointer
69  Electron Framework!ElectronMain [electron_library_main.mm : 31 + 0x5]
    rbp = 0x00007ffee8f66610   rsp = 0x00007ffee8f66590
    rip = 0x000000010d1d9214
    Found by: previous frame's frame pointer
70  Electron!main [electron_main.cc : 287 + 0xa]
    rbp = 0x00007ffee8f666c0   rsp = 0x00007ffee8f66620
    rip = 0x0000000106c9a110
    Found by: previous frame's frame pointer
71  libdyld.dylib!start + 0x1
    rbp = 0x00007ffee8f666d8   rsp = 0x00007ffee8f666d0
    rip = 0x00007fff20355631
    Found by: previous frame's frame pointer
72  libdyld.dylib!start + 0x1
    rbp = 0x00007ffee8f666d8   rsp = 0x00007ffee8f666d8
    rip = 0x00007fff20355631
    Found by: stack scanning
```

**Comment by deepak1556** [comment]
So the stack does indicate crash happens when trying to update the window title, was able to repro based on it. Steps

* Open new workspace that has no window title
* Open untitled editor
* Paste the provide text
* Window title will try to update to the text and crash

**Comment by deepak1556** [comment]
This seems to be fixed with our exploration builds based on Electron 11.

**Comment by nexxai** [comment]
Will this be explored as a fix in the current version or do I just need to wait for the public builds based on E11 to be released?

**Comment by deepak1556** [comment]
Electron 11 will be pushed to insiders next week, I don't plan to explore a fix for Electron 9.

**Comment by nexxai** [comment]
Thanks for confirming.  I will wait for the full release to contain E11.

**Comment by amanda-mitchell** [comment]
I found another string that will cause the editor to crash:

```
🚨 Possibly controversial opinions ahe 🚨 
```

Can anyone confirm whether this is fixed in the Electron 11 build? If it is not, then I will file a new bug.

**Comment by claui** [comment]
@amanda-mitchell It appears to be fixed in 1.53.0-insider.
Been able to reproduce it in earlier versions using the following string:

> `If you cut the blowfish wrong, you die.🐡`
